stdlib/2/__builtin__.pyi

```diff
@@ -736,6 +736,15 @@ def divmod(a: int, b: int) -> Tuple[int, int]: ...
 def divmod(a: float, b: float) -> Tuple[float, float]: ...
 def exit(code: Any = ...) -> NoReturn: ...
 @overload
+def filter(function: Callable[[AnyStr], Any],
+           s: AnyStr) -> AnyStr: ...
+@overload
+def filter(function: None,
+           t: Tuple[Any, ...]) -> Tuple[Any, ...]: ...
+@overload
+def filter(function: Callable[[Any], Any],
```

Comment:
And this one should have `Callable[[_T], _T]` for the callback and `_T` instead of `Any` in both places below.

Reply:
I think that should be `Callable[[_T], Any]`, to be consistent with the last implementation, but otherwise I stand corrected. I got confused because `Tuple[_T, ...]` looks like a single-type tuple to me, but I forgot about `Union`...